src/main/java/com/yelp/nrtsearch/server/cli/BackupWarmingQueriesCommand.java

```diff
@@ -46,11 +46,20 @@ public String getServiceName() {
     return serviceName;
   }
 
+  @CommandLine.Option(
+      names = {"-n", "--numQueriesThreshold"},
+      description = "Optional; minimum # of queries required to backup warming queries")
+  private int numQueriesThreshold;
+
+  public int getNumQueriesThreshold() {
+    return numQueriesThreshold;
+  }
+
   @Override
   public Integer call() throws Exception {
     LuceneServerClient client = baseCmd.getClient();
     try {
-      client.backupWarmingQueries(getIndex(), getServiceName());
+      client.backupWarmingQueries(getIndex(), getServiceName(), getNumQueriesThreshold());
```

Comment:
Can we add a uptime threshold as well? So we can fail if say the node hasn't been running for at least 3 hours